libgstc/pygstc/gstc.py

```diff
@@ -223,8 +224,7 @@ def _send_cmd_line(self, cmd_line):
         except Exception as exception:
             self._logger.error('%s error: %s' % (cmd,
                                                  type(exception).__name__))
-            traceback.print_exc()
```

Comment:
This gave unnecessary information to the user. Such as JSON Exception. The goal of this part is to precisely catch and handle these exceptions for the user.